portfolio/src/main/webapp/index.html

```diff
@@ -62,6 +62,10 @@ <h1>My Portfolio</h1>
         <p>Click here to get a random fact about me:</p>
         <button onclick="addRandomFact()">Fact!</button>
         <div id="fact-container"></div>
+        <div id="comments-container">
+          <p>Comments:</p>
+          <ul id="comments-list"></ul>
```

Comment:
should this be comments or comment-list instead?

Reply:
Done